includes/amp-helper-functions.php

```diff
@@ -171,6 +171,10 @@ function post_supports_amp( $post ) {
  * @return bool Whether it is the AMP endpoint.
  */
 function is_amp_endpoint() {
+	if ( is_admin() || is_feed() || ( defined( 'REST_REQUEST' ) && REST_REQUEST ) ) {
```

Comment:
We may need to revisit this with #1014 since we may want to render AMP content for a single property in the REST API response. We don't need to worry about it yet.